src/Paragraph/Paragraph.js

```diff
@@ -18,7 +18,13 @@ Paragraph.propTypes = {
   variant: PropTypes.oneOf(['ui', 'longForm', 'tiny']).isRequired
 };
 
-Paragraph.p = createWithComponent(Paragraph, 'p');
-Paragraph.span = createWithComponent(Paragraph, 'span');
+const createParagraph = el => ({ children, ...props }) => (
+  <Paragraph element={el} {...props}>
+    {children}
+  </Paragraph>
+);
+
+Paragraph.p = createParagraph('p');
```

Comment:
I wonder if `Paragraph.p` is even necessary given we can now pass `element` in?

Reply:
Good find, Paragraph.span is not needed either.